### ISSUE TITLE
NTR: Plasmodium falciparum PCR in blood positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8073]
New term request Plasmodium falciparum PCR in blood positive (samant_110129125347): Polymerase chain reaction; DNA

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000554
- label: Positive bloodstream plasmodium falciparum nucleic acid pathogen test
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of Plasmodium falciparum (a type of Malaria parasite) in the blood circulation by a method such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8073
- synonyms:
